hq/markdown/vulnerability-management.md

```diff
@@ -13,11 +13,11 @@ tasks that can't easily be automated, or we haven't got round to automating yet.
 There are 5 tools to focus on in this section. They are in rough order of priority. There are also some **critical vulnerabilities**
 which should always be addressed first. These are:
 
- - Unnecessarily exposed security groups (audit via [Security HQ](https://github.com/guardian/security-hq/blob/main/hq/markdown/vulnerability-management.md#1-security-hq))
- - IAM users with a password but no multi factor authentication (audit via [Security HQ](https://github.com/guardian/security-hq/blob/main/hq/markdown/vulnerability-management.md#1-security-hq))
- - IAM users with permanent credentials - these should be regularly rotated or deleted (audit via [Security HQ](https://github.com/guardian/security-hq/blob/main/hq/markdown/vulnerability-management.md#1-security-hq))
- - Critical vulnerabilities identified by [AWS Security Hub](https://github.com/guardian/security-hq/blob/main/hq/markdown/vulnerability-management.md#4-aws-security-hub)
- - Out of date AMIs (audit via [amiable](https://amiable.gutools.co.uk)) - some details [below](https://github.com/guardian/security-hq/blob/main/hq/markdown/vulnerability-management.md#2-operating-system-patches)
+ - Unnecessarily exposed security groups (audit via [Security HQ](https://github.com/guardian/security-hq/blob/main/hq/markdown/vulnerability-management#1-security-hq))
+ - IAM users with a password but no multi factor authentication (audit via [Security HQ](https://github.com/guardian/security-hq/blob/main/hq/markdown/vulnerability-management#1-security-hq))
+ - IAM users with permanent credentials - these should be regularly rotated or deleted (audit via [Security HQ](https://github.com/guardian/security-hq/blob/main/hq/markdown/vulnerability-management#1-security-hq))
+ - Critical vulnerabilities identified by [AWS Security Hub](https://github.com/guardian/security-hq/blob/main/hq/markdown/vulnerability-management#4-aws-security-hub)
+ - Out of date AMIs (audit via [amiable](https://amiable.gutools.co.uk)), [operating systems patches](https://github.com/guardian/security-hq/blob/main/hq/markdown/vulnerability-management#2-operating-system-patches)
```

Comment:
These links don't work for me without the `.md`

Reply:
I haven't tested this with the app locally, i'll chase down what's up.

Reply:
This behaves as expected in the app, but as you say not in the GitHub UI. I think the proper way to fix this is probably to make the app serve the files with `.md` appended.

Reply:
This change should fix the routing: https://github.com/guardian/security-hq/pull/457/commits/5d366e2cefa2703f937a062086b2248f5bd23747